emission/net/ext_service/otp/otp.py

```diff
@@ -112,116 +107,19 @@ def turn_into_trip(self, _id, user_id, trip_id, is_fake=None):
         final_start_time = otp_time_to_ours(our_json['plan']['itineraries'][0]["startTime"])
         final_end_time = otp_time_to_ours(our_json['plan']['itineraries'][0]["endTime"])
         cost = 0
-
-        #print start_city, end_city
```

Comment:
I got rid of those variables, they were for my hacky wrong traffic detection I tried